src/osm_configurator/view/toplevelframes/reduction_frame.py

```diff
@@ -14,13 +15,13 @@ class ReductionFrame(TopLevelFrame):
     calculate the Reduction. The other frame provides the default calculation-values.
     """
 
-    def __init__(self, state_manager, control):
+    def __init__(self, state_manager, category_controller):
```

Comment:
Please include typehints

Reply:
The Frames are not part of this branch, so their typehints will be done in another branch